target/linux/imx/cortexa53/target.mk

```diff
@@ -2,6 +2,7 @@ ARCH:=aarch64
 BOARDNAME:=NXP i.MX with Cortex-A53 (ARM64)
 CPU_TYPE:=cortex-a53
 KERNELNAME:=Image dtbs
+FEATURES:=display
```

Comment:
We don't want to change this, it is already set (at subtarget level): https://github.com/openwrt/openwrt/blob/96978da1d9d83b0bcee67fb8c948daacf8c3092c/target/linux/imx/Makefile#L9 that's why we saw the CI issue. We want to change the package definition for `fb`